### PR TITLE
remove tahoe from render settings

### PR DESCRIPTION
**Pull request by V-Gorash**
### PURPOSE
Legacy Tahoe plugin is in render settings

### EFFECT OF CHANGE
Now Tahoe has been removed from settings

### NOTES FOR REVIEWERS
This PR makes Tahoe fully unavailable! Other Tahoe code will be deleted later

### CODE REVIEW

File: pxr/imaging/plugin/hdRpr/rprApi.cpp [hunk 1]
```diff
@@ -1690,9 +1690,7 @@ class HdRprApiImpl {
 
             if (m_state == kStateRender && config->IsDirty(HdRprConfig::DirtyRenderQuality)) {
                 TfToken activeRenderQuality;
```
Comment (V-Gorash) [inline pxr/imaging/plugin/hdRpr/rprApi.cpp:1692]: After remove "Full" setting from generateRenderSettingFiles.py token HdRprCoreRenderQualityTokens->Full is unavailable

File: pxr/imaging/plugin/hdRpr/rprApi.cpp [hunk 2]
```diff
@@ -3377,9 +3375,7 @@ Don't show this message again?
 
 private:
     static RprUsdPluginType GetPluginType(TfToken const& renderQuality) {
-        if (renderQuality == HdRprCoreRenderQualityTokens->Full) {
```
Comment (V-Gorash) [inline pxr/imaging/plugin/hdRpr/rprApi.cpp:3380]: fter remove "Full" setting from generateRenderSettingFiles.py token HdRprCoreRenderQualityTokens->Full is unavailable

File: pxr/imaging/plugin/hdRpr/rprApi.cpp [hunk 3]
```diff
@@ -1690,9 +1690,7 @@ class HdRprApiImpl {
 
             if (m_state == kStateRender && config->IsDirty(HdRprConfig::DirtyRenderQuality)) {
                 TfToken activeRenderQuality;
-                if (m_rprContextMetadata.pluginType == kPluginTahoe) {
-                    activeRenderQuality = HdRprCoreRenderQualityTokens->Full;
```
Comment (V-Gorash) [inline pxr/imaging/plugin/hdRpr/rprApi.cpp:1694]: After removing "Full" setting from generateRenderSettingFiles.py token HdRprCoreRenderQualityTokens->Full is unavailable

**Comment by ARozghon** [comment]
worked as expected

**Review by s1lentssh — APPROVED** [review]

**Review by Vsevolod1983 — APPROVED** [review]

**Pull request merged**